src/github.rs

```diff
@@ -1750,6 +1758,42 @@ impl<'q> IssuesQuery for Query<'q> {
                         bot_tracking_comment_content,
                         initiating_comment_html_url: init_comment.html_url.clone(),
                         initiating_comment_content: quote_reply(&init_comment.body),
+                        disposition: fcp
+                            .fcp
+                            .disposition
+                            .as_deref()
+                            .unwrap_or("<unknown>")
+                            .to_string(),
+                        pending_reviewers: fcp
```

Comment:
TIL using the Zulip ID for pings, that's a neat trick! 

Though I think in this context we don't want to emit ping every week, I'm afraid it will just become annoying. I would rather instead remove the `@` mentions and just list the people the FCP is waiting on.

Reply:
We may not want to ping every week but I think we should ping from time to time, and I think pinging T-compiler member in the meeting at the specific things waiting for them is acceptable to me.

Especially considering that they are already being pinging at the start of the meeting.

Reply:
After some careful thinking I think I may have found a compromise.

I propose that we only mention under those circumstance:
  - we are 2 weeks into the FCP *(even if the are concerns registered, so that team members can voice there opinions on the FCP and concerns)*
  - or when there are no concerns and we are at least 4 weeks into the FCP *(no concerns + long after the start: probably didn't see the ping or forgot; and if the are not happy about the proposal they can just put up a concern)*

Btw, I used your `_` trick to do the no-but-still-looking-mention.